### DISCUSSION TITLE
Add TempDB to integrity check

### DISCUSSION THREAD

**Discussion by FrenchDataWizard** [olahallengren/sql-server-maintenance-solution#394] [Questions & Answers]
As part of the system DBs, why not run DBCC CheckDB on TempDB as well?

**Comment by olahallengren** [comment]
Tempdb is recreated when you restart SQL Server, and does not contain any data. This is the reason, I am not including it in SYSTEM_DATABASES.

If you still prefer to perform checkdb of tempdb, you can do like this:

```
EXECUTE dbo.DatabaseIntegrityCheck
@Databases = 'SYSTEM_DATABASES,tempdb',
@CheckCommands = 'CHECKDB'
```
